src/Native/Unix/System.Security.Cryptography.Native/opensslshim.c

```diff
@@ -27,10 +28,27 @@ FOR_ALL_OPENSSL_FUNCTIONS
 
 // x.x.x, considering the max number of decimal digits for each component
 #define MaxVersionStringLength 32
-#define SONAME_BASE "libssl.so."
 
 static void* libssl = NULL;
 
+#ifdef __APPLE__
+#define DYLIBNAME_PREFIX "libssl."
+#define DYLIBNAME_SUFFIX ".dylib"
+#define MAKELIB(v) DYLIBNAME_PREFIX v DYLIBNAME_SUFFIX
+#else
+#define SONAME_BASE "libssl.so."
+#define MAKELIB(v)  SONAME_BASE v
+#endif
+
+static void DlOpen(const char* libraryName)
+{
+    assert(libssl == NULL);
+    libssl = dlopen(libraryName, RTLD_LAZY);
+#ifdef DEBUG_LIBRARY_LOAD
+    printf("Attempting load of %s ... %s\n", libraryName, libssl == NULL ? "failed." : "succeeded!");
```

Comment:
A nit - I wonder if it is worth keeping this in

Reply:
I'm not sure if that's "always do it for debug builds" or "it's a small enough thing to add back later, let's just go ahead and delete the code".  (I'm fine either way)

Reply:
I would just delete it, it is easy enough to add if needed at some point later.